src/constants/chains.ts

```diff
@@ -87,6 +87,16 @@ export const TESTNET_CHAIN_IDS = [
   SupportedChainId.OPTIMISM_GOERLI,
 ] as const
 
+export const TESTNET_SET = new Set([
```

Comment:
wait why add a new one here with more testnets instead of using existing `TESTNET_CHAIN_IDS`? seems like these are out of sync now?